data/hai/hai missions.txt

```diff
@@ -1658,3 +1658,117 @@ mission "Strider notification"
+	minor
+	source "Greenwater"
+	destination "Darkrest"
+	passengers 5
+	to offer
+		random < 30
+		has "First Contact: Hai: offered"
+	on offer
+		conversation
+			`It is a sunny afternoon as you idly walk through the spaceport, until you somehow find yourself surrounded by a group of young humans and Hai.`
+			`	"You're a spaceship captain, right? Could you take us to <planet>?" says one of the Hai, as if he had practiced this line for a while now.`
+			`	The human next to him gives him a nudge. "Don't forget to ask for a trip back, Tagreep, unless you want to be stranded there."`
+			choice
+				`	"What would you want to do on <planet>? Isn't that an abandoned station?"`
+					goto question
+				`	"Sure, I can take you there and back."`
+					goto accept
+				`	"How much can you pay me?"`
+				`	"Sorry, I'm busy right now. Go ask some other captain."`
+						decline
+			`	The group starts to shuffle around with a nervous air. "Well, we're still in school, so money's kind of tight right now... But we can pay you! Is <payment> okay?" This last part is said in almost a whisper.`
+			choice
+				`	"Why would you want to pay money to go to <planet>, anyhow? Isn't that an abandoned station?"`
+					goto question
+				`	"Sure, I'm not hurting for cash right now."`
+					goto accept
+				`	"No way, that's too low. Find someone else."`
+						decline
+			label question
+			`	"Well, that's kind of why we want to go there," says one of the Hai. "Some guy in our dorm told us that there's ghosts haunting <planet>, and we wanted call his bluff."`
+			`	"We're planning to look around for an afternoon, see what we can find," one of the human students says, grinning. "Once we're done we'll let you know, and then we can leave."`
+			`	One of the shorter Hai in the back pipes up. "I brought cameras for the trip!" He is ignored by the rest of the group.`
+			choice
+				`	"Is this legal?"`
+				`	"Not my business where you spend your vacations. Come aboard."`
+					goto accept
+			`	The students look at each other, considering this question for what is apparently the first time. "We've heard of other people going there with no problems, as long as they were... discreet about it," says one of the students, trying to assure both you and their fellow classmates. Another mentions having heard something about needing a permit to enter, but he is quickly shushed by the rest of the group.`
+			choice
+				`	"I'm not about to risk getting in trouble for this. Find someone else."`
+						decline
+				`	"Not my business where you spend your vacations. Welcome aboard."`
+			label accept
+			`	The students race onboard your ship. As they settle into their bunks, you can hear them speculating about what they will find on <planet>.`
+				accept
+	on visit
+		dialog `You look for your passengers, but realize that they took a ride on one of your escorts! Better depart and wait for your escorts to arrive in this star system.`
```

Comment:
I think there is a generic version of this available as a phrase
You could have just launched and landed back bc you forgot smth and get this msg btw

Reply:
Made the phrase a little less generic.